<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AQTelemetryView</class>
 <widget class="QWidget" name="AQTelemetryView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="tab_val_chart">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="plotFrameTele">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>589</width>
        <height>405</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_start_tel_grid">
       <property name="text">
        <string>Start Telemetry</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
